polaris-icons/icons/PersonalizedTextMajor.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you run this through an optimizer like https://jakearchibald.github.io/svgomg/ ?

Reply:
done